src/module/interface/timeline/index.ts

```diff
@@ -7,6 +7,7 @@ export interface ITimeline {
   ventureId: string;
   userId?: string;
   membersWrite: boolean;
+  lastUpdate?: string;
```

Comment:
New metadata property returned by the `apiserver` (in https://github.com/venturemark/apiserver/pull/235) which indicates the ID of the most recent update in the timeline.